.goreleaser.yaml

```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
+      - "--platform=linux/amd64"
+    image_templates:
+      - "ghcr.io/persona-id/{{.ProjectName}}:{{ .Tag }}-amd64"
+    goarch: arm64
```

Comment:
I think `goarch` should be amd64 or a reference to the x86 arch?

Reply:
Yeap, I must have copied back because this was the original block.